.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Test
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "*" ]
+
+  workflow_dispatch:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
```

Comment:
Added  `distribution `

     `   with:
          distribution: 'oracle'
          java-version: '17'`

Reply:
yes you are right - thats why my first action failed